version/version.go

```diff
@@ -13,13 +13,21 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
+	// The next version number that will be released. This will be updated after every release
+	// Version must conform to the format expected by github.com/hashicorp/go-version
+	// for tests to work.
+	// A pre-release marker for the version can also be specified (e.g -dev). If this is omitted
 	// The main version number that is being run at the moment.
-	Version = "1.8.6"
+
+	Version = "1.8.5"
```

Comment:
Dev version should be the next version; matching the contents of the new version/VERSION file. 
```suggestion
	Version = "1.8.6"
```